packages/flare/bin/cron_job_ingest_events.py

```diff
@@ -161,15 +161,21 @@ def save_start_date(app: client.Application, tenant_id: int) -> None:
             value=date.today().isoformat(),
         )
 
-    # If the current tenant has changed, update the start date so that future requests will be based off today
+    # If the tenant has changed, update the start date so that future requests will be based off today
     # If you switch tenants, this will avoid the old tenant from ingesting all the events before today and the day
     # that tenant was switched in the first place.
-    if current_tenant_id != tenant_id:
+    if last_ingested_tenant_id != tenant_id:
         app.service.kvstore[KV_COLLECTION_NAME].data.update(
             id=CollectionKeys.START_DATE.value,
             data=json.dumps({"value": date.today().isoformat()}),
         )
 
+        save_collection_value(
+            app=app,
+            key=CollectionKeys.LAST_INGESTED_TENANT_ID.value,
+            value=tenant_id,
+        )
+
```

Comment:
Is this meant to be in the if block? Should be outside, no?

Reply:
I've put it in for the tests to see if it actually changed or not, otherwise it doesn't really matter since the value hasn't changed